### PR TITLE
docs: [IAC-1999]: Add connector as part of IaCM onboarding

### PR DESCRIPTION
**Pull request by richardblack-Harness**
## Description

* Tabbed section regarding adding Cloud Provider and Code Repo connectors included in the IaCM onboarding doc
* Tabs include two interactive walkthroughs and a written step by step guide.
* [JIRA](https://harness.atlassian.net/browse/IAC-1999)

### Screenshot / Preview
AddConnectorTabs

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/infra-as-code-management/get-started/onboarding-guide.md [hunk 1]
```diff
@@ -59,6 +53,93 @@ resource "aws_instance" "my_first_ec2_instance" {
...
+5. Continue to the final step to confirm your Cloud Provider connection is verified successfully.
+6. Select **Finish**
+
+ Go to [Connect your Cloud Provider](https://developer.harness.io/docs/category/cloud-providers) for more information regarding connecting your Cloud Provider.
+
+### Add a Git repository connector
+
+Using GitHub as an example, after following the initial five steps to reach the Connectors panel:
+
+1. Select **GitHub**
```
Comment (SmritiSatya) [inline docs/infra-as-code-management/get-started/onboarding-guide.md:123]: Period at the end of every step (global).

File: docs/infra-as-code-management/get-started/onboarding-guide.md [hunk 2]
```diff
@@ -59,6 +53,93 @@ resource "aws_instance" "my_first_ec2_instance" {
...
+
+When adding any connector, start by:
+
+1. Signing in to [app.harness.io](https://app.harness.io).
+2. In the module pane, select **Infrastructure**.
+3. Select **Project Setup**, and then select **Connectors**
+4. Select **New Connector**
+5. Select the appropriate Connector, e.g. Cloud Provider, Code Repository
+
+### Add a Cloud Provider connector
```
Comment (SmritiSatya) [inline docs/infra-as-code-management/get-started/onboarding-guide.md:101]: Add a cloud provider connector

File: docs/infra-as-code-management/get-started/onboarding-guide.md [hunk 3]
```diff
@@ -59,6 +53,93 @@ resource "aws_instance" "my_first_ec2_instance" {
 
 Go to [Terraform Documentation](https://developer.hashicorp.com/terraform/intro) or [OpenTofu Documentation](https://opentofu.org/docs/) for more information on currently supported Workspace types. 
 
+## Add connectors
+
+Add Cloud Provider and Git Connectors to select when Creating a workspace. Doing so allows your Cloud Provider and Git repository to read and write changes applied via your Harness IaCM pipelines.
+
+:::info
+We recommend configuring your connector before creating your workspace, however, you can also add new connectors during the Create Workspace flow.
```
Comment (SmritiSatya) [inline docs/infra-as-code-management/get-started/onboarding-guide.md:61]: Can we provide an internal link to the "create workspace flow"?

**Review by SmritiSatya — DISMISSED** [review]

File: docs/infra-as-code-management/get-started/onboarding-guide.md [hunk 4]
```diff
@@ -59,6 +53,93 @@ resource "aws_instance" "my_first_ec2_instance" {
 
 Go to [Terraform Documentation](https://developer.hashicorp.com/terraform/intro) or [OpenTofu Documentation](https://opentofu.org/docs/) for more information on currently supported Workspace types. 
 
+## Add connectors
+
+Add Cloud Provider and Git Connectors to select when Creating a workspace. Doing so allows your Cloud Provider and Git repository to read and write changes applied via your Harness IaCM pipelines.
+
+:::info
+We recommend configuring your connector before creating your workspace, however, you can also add new connectors during the [Create Workspace flow](https://developer.harness.io/docs/infra-as-code-management/get-started/onboarding-guide#create-a-workspace).
```
Comment (brian-f-harness) [inline docs/infra-as-code-management/get-started/onboarding-guide.md:61]:
```suggestion
Harness recommends configuring your connector before creating your workspace, however, you can also add new connectors during the [Create Workspace flow](https://developer.harness.io/docs/infra-as-code-management/get-started/onboarding-guide#create-a-workspace).
```

File: docs/infra-as-code-management/get-started/onboarding-guide.md [hunk 5]
```diff
@@ -59,6 +53,93 @@ resource "aws_instance" "my_first_ec2_instance" {
...
+1. Signing in to [app.harness.io](https://app.harness.io).
+2. In the module pane, select **Infrastructure**.
+3. Select **Project Setup**, and then select **Connectors**.
+4. Select **New Connector**.
+5. Select the appropriate Connector, e.g. Cloud Provider, Code Repository.
+
+### Add a cloud provider connector
+Using AWS as an example, after following the initial five steps to reach the Connectors panel:
+
+1. Select **AWS**
```
Comment (brian-f-harness) [inline docs/infra-as-code-management/get-started/onboarding-guide.md:104]:
```suggestion
1. Select **AWS**.
```

**Comment by richardblack-Harness** [comment]
New PR open for this one: https://github.com/harness/developer-hub/pull/6815

**Pull request closed without merging**